### ISSUE TITLE
Environment variable UPDATE is too generic

### ISSUE THREAD

**Issue by dwiajik** [wellguimaraes/mocha-snapshots#6]
I think using environment variable `UPDATE` for updating snapshot is somewhat dangerous. It is because the name is too generic, user might have set the env var for another purpose. I think it would be better to use more detailed/specific env variable name, e.g. `MOCHA_SNAPSHOT_UPDATE`. Or even just provide only `--update` as the only way to update the snapshot.

Just my two cents, good work! :+1:

**Comment by wellguimaraes** [comment]
Makes sense. Maybe having only `--update` is the best choice for next version. ;)
